grpc-api/src/test/app.d/01-groovy.groovy

```diff
@@ -0,0 +1,7 @@
+import io.deephaven.appmode.ApplicationContext
+import io.deephaven.db.tables.utils.TableTools
```

Comment:
this should be unnecessary, right? or are we not doing "free" imports any more?

probably should write the kind of code we expect to have users write.

--

also, would it not make sense to have the AppContext import and app = AppContext.get() binding already done for you for these scripts? since any script that does anything interesting will need to do it anyway..

Reply:
I'd like to get rid of "free" imports here where we have the chance to do something.

Also, I don't like implicit bindings that scripts just "know" about. Often times, I'd see PQs at customers where they would do:

```groovy
db = (Database)db
```
just so they would have strong typing. I'd prefer to create a pattern where scripts are explicit.

Alternatively, we can have a pattern where instead of a "script" the scripts provide a function with a known signature:

```groovy
def script_eval(app):
   ...
```
But that feels a bit clunkier.